src/screens/calendar/components/EventsList.tsx

```diff
@@ -41,7 +41,7 @@ export const EventsList = React.forwardRef<FlatList, EventsListProps>(
 
     const { offset } = getItemLayout(days, currentIndex)
 
-    const btnShownCondition = pageOffsetY !== offset
+    const btnShownCondition = Math.abs(pageOffsetY - offset || offset - pageOffsetY) > 300
```

Comment:
if you use Math.abs, pageOffsetY - offset and offset - pageOffsetY will probably always return the same value and second part in parentheses will never be executed anyway

Reply:
You're right, the result is in fact the same.